routers/api/v1/api.go

```diff
@@ -642,11 +642,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 					}, reqGitHook(), context.ReferencesGitRepo(true))
 				}, reqToken(), reqAdmin())
 				m.Group("/collaborators", func() {
-					m.Get("", repo.ListCollaborators)
+					m.Get("", reqAnyRepoReader(), repo.ListCollaborators)
 					m.Combo("/:collaborator").Get(repo.IsCollaborator).
```

Comment:
```suggestion
					m.Group("/:collaborator"). func() {
					    m.Get(reqAnyRepoReader(), repo.IsCollaborator).
```

Reply:
@davidsvantesson I have made the change.